src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -168,6 +167,19 @@ public static void main(final String[] args) {
     }
   }
 
+  private static JFrame newMainFrame() {
+    final JFrame frame = new JFrame("TripleA");
+
+    frame.add(new MainPanel(setupPanelModel));
+    frame.pack();
+
+    frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
This is the core of the fix.  Changed from `DISPOSE_ON_CLOSE` to `EXIT_ON_CLOSE`.